tool/templates/cpp/enum_decl.h.jinja

```diff
@@ -27,7 +27,7 @@ public:
 
 	{% if !m.docs.is_empty() -%}
 	/**
-	 * {{m.docs|indent_trimmed(2)}}
+{{~ m.docs|prefix_trimmed("	 * ")}}
```

Comment:
Issue: we should try and keep the templates correctly indented so that they are readable.

Reply:
It's not always possible, sometimes (like here) you need to eat only the whitespace on that line

Reply:
iiuc if you use `{{~` it will strip all whitespace except the newline, so you should be able to indent this without a behaviour change